### DISCUSSION TITLE
AVI metadata not used in Photoprism

### DISCUSSION THREAD

**Discussion by Daniel-H123** [photoprism/photoprism#3621] [Q&A]
Hey everyone,

I've just set up my first PhotoPrism env and uploaded some photos and videos and noticed some dates displaying unknown. The problem occurs only for me with AVI files. I know that it has to do with the support of EXIF with AVI files, but what can I do to let PhotoPrism take the File Created date or something like that. It would be a pain to change the date on all my older video's manualy.

**Comment by graciousgrey** [comment]
Hey :wave: 

PhotoPrism reads metadata using exiftool. If AVI metadata is not supported by exiftool, that is why PhotoPrism cannot read it at the moment.

https://docs.photoprism.app/user-guide/library/metadata/

If no information is found in the metadata or the original names, the file system's mod time is used to sort the files (e.g. when sorting by "newest first"). However, this is usually not the actual date a photo was taken (or a graphic was created by the original author), but only the time you downloaded or copied it. Therefore, we do not set it as the TakenAt date.

**Comment by Daniel-H123** [comment]
Thanks for the answer.

What I am experiencing, while importing old sd-cards is that all files with the exif metadata are placed in the correct folderstructure, like: /2000/01/photo.jpg, but the .avi's with no metadata (but with the correct mod timestamp) are placed in the 2023 folder. This is because the mod time is not transferred when the file is copied from the uploads folder. This is not how it is supposed to work right?

Edit: The correct folder is used when using import via the import folder, but not when using the upload button in the UI. The sort method only works with the new mod time (when the file whas imported/copied) when using both methods.

The problem in my case is that it does not keep the original file mod timestamp I think.

**Reply by graciousgrey** [reply]
If you send us a sample file to samples@photoprism.app, I can take a closer look :)
